Ask question page

gostrobrod opened the pull request:
Layout for Ask question page

wenpingdu committed:
Create a template for AskQuestion Page

wenpingdu committed:
Build basic layout for AskQuestion page

gostrobrod committed:
Stub of AskQuestionViewModel

gostrobrod committed:
Use own question card layout in AskQuestion

gostrobrod committed:
Remove selecting a question in askQuestion.

wenpingdu committed:
Reuse ReadingViewModel and Open Keyboard by default for AskQuestion page

wenpingdu committed:
Screen adjust when keyboard appearing for Ask question page

gostrobrod committed:
Change back button icon and 'Ask Question' to 'Write Question'

wenpingdu committed:
set WindowSoftInputModeAdjust in OnStart() of App

gostrobrod committed:
Copies styles in Write Question Page

gostrobrod committed:
Rename Ask Question Page to Write Question Page

gostrobrod committed:
Fix styles and add separator to Write Question page

gostrobrod committed:
Keyboard editor field card for Write Question page

wenpingdu committed:
Modify Editor Container layout

gostrobrod committed:
Merge branch 'main' into Feature/AskQuestion_page

wenpingdu committed:
Add Character Counter for Editor

wenpingdu committed:
Modify Android Editor's style and Button's style

gostrobrod committed:
Add accessebility for navigation items in Write Question Paga

gostrobrod committed:
Merge branch 'main' into Feature/AskQuestion_page

wenpingdu committed:
fix iOS keyboard issue for screen adjust

gostrobrod committed:
Label layout in WriteQuestionPage

gostrobrod committed:
Fix safe area problem on iOS in Write Question Page

gostrobrod committed:
Clear button in editor field in Write Question Page

gostrobrod committed:
Styles of clear button in editor field in Write Question Page

gostrobrod committed:
Accessibility labels for clear button in editor field in Write Question Page

wenpingdu committed:
Overall styles in Write question page

gostrobrod committed:
Remove AskQuestionPreviewCard

vteague reviewed (approved):
Looks great, thanks.

vteague committed:
Merge branch 'main' into Feature/AskQuestion_page

# Conflicts:
#	RightToAskClient/RightToAskClient/RightToAskClient/Views/AppShell.xaml.cs

vteague committed:
Change Android statusbar colour.

vteague committed:
Moved strings into AppResources.Designer.cs.
Edited styles to deal with dark mode.
Removed explicit background colour setting - it now defaults to the same as BaseView, which matches the mode (dark or light).
